src/getting-started/create-bootable-media/bootable-usb-using-macos.md

```diff
@@ -1 +1,99 @@
+- [Flash the ISO to your USB storage device.](/Getting-Started/Create-bootable-media/bootable-usb-using-macos.html#create-the-bootable-usb)
+
+---
+
+## Install Etcher
+
+1. Navigate to Etcher's [download page](https://www.balena.io/etcher/) and click `Download for macOS`.
+
+![Download Etcher](/images/create-bootable-usb-macos/download-etcher.png)
+
+2. Click the download folder from the Dock and select the Etcher .dmg file.
+
+![Run Etcher dmg](/images/create-bootable-usb-macos/run-etcher-dmg.png)
+
+3. Drag the Etcher logo into the applications folder.
+
+![Install Etcher](/images/create-bootable-usb-macos/install-etcher.png)
+
+## Download the Pop!_OS.iso File
+
+![Linux Download ISO](/images/create-bootable-usb-linux/using-linux-download-iso.png)
+
+1. Download the appropriate Pop!_OS.iso file for your target system. See the [preparation guide](/Getting-Started/Create-bootable-media/create-bootable-usb.html#choose-a-pop_os-image) for information about each version.
+
+2. Take note of the hash value for your chosen ISO. Use this value to verify the integrity of the ISO file.
+
+![Compare Hash Values](/images/create-bootable-usb-linux/compare-hash-values.png)
+
+3. If prompted, confirm you want to allow downloads from pop.system76.com.
+
+## Verify the Hash Value
+
+1. Click the Launchpad icon, then search for `terminal`. Click the `Terminal` icon to launch it.
+
+![Launch Terminal](/images/create-bootable-usb-macos/launch-terminal.png)
+
+2. Change directory to the folder containing your Pop!\_OS ISO file. In this example the ISO is in the Downloads folder.
+
+```bash
+cd Downloads
+```
+
+3. Use the `shasum` command to verify your ISO file. Make sure the name of the ISO matches exactly what is in your folder. The filename changes depending on the ISO version you chose.
+
+```bash
+shasum -a 256 pop-os_21.10_amd64_intel_8.iso
+```
+
+4. This command will output a long string of characters. Compare the output to the appropriate hash value on the Pop!\_OS download site.
+
+## Create the Bootable USB
+
+1. After installation, click the Launchpad icon on the Dock. Select the `balenaEtcher` icon to launch the app. If prompted, confirm you want to open the app.
+
+![Launch Etcher](/images/create-bootable-usb-macos/launch-etcher.png)
+
+2. Select `Flash from file`, then navigate to and select your Pop!\_OS.iso file.
+
+![Flash From File](/images/create-bootable-usb-macos/flash-from-file.png)
+
+3. Click `Select target` and select your USB drive.
+
+![Select Target](/images/create-bootable-usb-macos/select-target.png)
+
+4. Click `Flash!`. Enter your password if prompted.
+
+![Click Flash](/images/create-bootable-usb-macos/click-flash.png)
+
+5. Close Etcher once the installation completes.
+
+![Click Finish](/images/create-bootable-usb-macos/click-finish.png)
+
+## Next Steps
+
+Use your bootable Pop!_OS USB to demo, install, or recover a current Pop!_OS installation.
+
+### Install Pop!_OS
```

Comment:
Since this part isn't OS-specific, is there a reason this needs to be duplicated between the three OS-specific pages instead of just being at the top of the installation page?

Reply:
Also, it might be good to clarify if this table is specific to System76 hardware (and maybe suggest the user consult their hardware manufacturer's documentation if they're using third-party hardware).

Reply:
- This was done to functionally segment the USB creation guides from the Installation guide. It's not OS-specific, but is always necessary information when using a live USB. I had intended for the Installation guide to be agnostic of any installation method (although it does suggest the USB method under the "Installation Media" requirement). 99% of users will use the USB method so I'll do as you suggest. 

- I will clarify this when making the above mentioned changes